espnet2/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.9.6"
```

Comment:
We put the version information in https://github.com/espnet/espnet/blob/master/setup.py#L126, and we may forget to update the two places.
Is it possible to automatically get the version information from https://github.com/espnet/espnet/blob/master/setup.py#L126?

If this way is necessary, we'll note it when releasing the new version.

Reply:
it might be better to do it in the reverse direction, i.e. maintain this version here (which is accessible super easily from the code) and get it from setup.py.

That's what we're doing in https://github.com/huggingface/huggingface_hub/blob/62b0ddbc1fb5d9f7c29872332b49bd6d922aac8e/setup.py#L4-L11

What do you think?

Reply:
I agree this modification. 
Since the current version is not updated from the installation even if we use editable installation.

Reply:
We changed version location.
https://github.com/espnet/espnet/pull/2840

Reply:
@kan-bayashi Do you want me to align this PR on the change from #2840 or will you do it?

Reply:
I will fix confilct 👍